test/api/news.test.ts

```diff
@@ -0,0 +1,95 @@
+import { getNewsHandler, postNewsHandler} from '../../pages/api/news';
+import httpMocks, { Body } from 'node-mocks-http';
+import prisma from '../../prisma/prisma';
+import { News } from '@prisma/client';
+import { NewsWrite } from '../../lib/api/createNewsRequest.dto';
+import { BadRequestError } from '../../lib/utils/errors/badRequestError';
+
+const createNews0:NewsWrite = {
```

Comment:
📌 We might want these in an examples or fixtures folder

Reply:
@varunthakkar1 would we want an examples folder- where in the file directory would be best?

Reply:
@jessicasu77 test/api/data/newsFixtures.ts maybe? or maybe test/api/fixtures, what do you think @rymaju

Reply:
@varunthakkar1  yup we put it in /test/api/examples/newsMockData.ts